websites/P/poke/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"name": "ashley",
+		"id": "396571938081865741"
+	},
+	"service": "Poke",
+	"description": {
+		"en": "Watch Silly videos or search the interwebs while being anonymous on poke!"
+	},
+	"url": [
+		"poketube.fun",
+		"poke.ggtyler.dev",
+		"poke.vern.cc"
+	],
+	"version": "1.0.0",
```

Comment:
There is a new field called `matches`, which is required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.